ironfish/src/primitives/transaction.ts

```diff
@@ -92,6 +92,10 @@ export class Transaction {
     })
   }
 
+  isMinersFee(): boolean {
+    return this.spendsLength() === 0 && this.notesLength() === 1
```

Comment:
oh I think we can also check if the transactionFee() is negative too actually

Reply:
so the condition should be:

```
spendsLength == 0 AND
notesLength == 1 AND
transactionFee <= 0

```
right?